packages/components/dialog/src/Dialog.stories.tsx

```diff
@@ -154,18 +172,14 @@ export const HTMLForm = () => {
             </Dialog.Header>
 
             <Dialog.Body className="flex flex-col gap-lg">
-              {!isAccountCreated ? (
```

Comment:
@Powerplex I simplify this example so they can just copy/paste as placeholder. WDYT ? 🙏🏻